File: src/assets-en/courses/xml/gyak06_start/src/org/iit/xml/feladat2/Car.java
package org.iit.xml.feladat2;

public class Car {
	
	private String manufacturer;
	private String color;
	private double price;
	
	public Car(String manufacturer, String color, double price) {
		this.manufacturer = manufacturer;
		this.color = color;
		this.price = price;
	}
	
	public Car() {
		this("", "", 0);
	}
	
	public String getManufacturer() {
		return manufacturer;
	}
	
	public void setManufacturer(String manufacturer) {
		this.manufacturer = manufacturer;
	}
	
	public String getColor() {
		return color;
	}
	
	public void setColor(String color) {
		this.color = color;
	}
	
	public double getPrice() {
		return price;
	}
	
	public void setPrice(double price) {
		this.price = price;
	}
	
	@Override
	public String toString() {
		return String.format("Car[manufacturer=%s, color=%s, price=%f]", manufacturer, color, price);
	}
	
}

File: src/assets-en/courses/db-msc/gyak05/LINQ/Model/Car.cs
namespace LINQ.Model
{
    public class Car
    {
        public string PlateNumber { get; set; }

        public string Manufacturer { get; set; }

        public decimal Price { get; set; }

        public string Owner { get; set; }

        public override string ToString()
        {
            return string.Format("Car[PlateNumber = {0}, Manufacturer = {1}, Price = {2}, Owner = {3}]", PlateNumber, Manufacturer, Price, Owner);
        }
    }
}
